frontend/taipy-gui/src/components/Taipy/Progress.tsx

```diff
@@ -110,7 +118,7 @@ const Progress = (props: ProgressBarProps) => {
 
     return showValue && value !== undefined ? (
         linear ? (
-            <Box sx={boxWithFlexDirectionSx}>
+            <Box sx={boxWithFlexDirectionSx} data-testid="linear-progress-container">
```

Comment:
Do we really need this attribute??

Reply:
This is used only for unit tests only, making it more stable than the auto-generated className from React, which might change due to styling or refactoring

Reply:
Sure. But then.... all application get this... I mean users
Can you not add the class name when generating the component when testing (yes you can)?